common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIDProviderConfiguration.java

```diff
@@ -30,6 +30,7 @@
  * https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata
  */
 
+@SuppressWarnings("PMD")
```

Comment:
What kind of PMD warning was suppressed here?

Reply:
Just thinking out loud: we should try to provide context/comments for these where it's not super-clear what's being flagged

Reply:
I have added comments describing the suppression. Thanks for the suggestion Brian